fedimint-cli/src/ng.rs

```diff
@@ -82,6 +97,83 @@ pub async fn handle_ng_command<D: IDatabase>(
 
             Ok(serde_json::to_value(amt).unwrap())
         }
+        ClientNg::LnPay { bolt11 } => {
+            let invoice = bolt11.clone();
+            let operation_id = bolt11.payment_hash().into_inner();
+            let ln_client = client
+                .get_module_client::<LightningClientModule>(LEGACY_HARDCODED_INSTANCE_ID_LN)
+                .unwrap();
+            let mut dbtx = client.db().begin_transaction().await;
+            let outgoing_ln_contract = ln_client
+                .create_outgoing_ln_contract(
+                    &mut dbtx.with_module_prefix(LEGACY_HARDCODED_INSTANCE_ID_LN),
+                    client.context(),
+                    invoice,
+                    rng,
+                )
+                .await?;
+
+            let (contract_id, amount) = match &outgoing_ln_contract {
+                LightningOutput::Contract(c) => {
+                    let contract_id = c.contract.contract_id();
+                    let amount = c.amount;
+                    (contract_id, amount)
+                }
+                LightningOutput::Offer(_) | LightningOutput::CancelOutgoing { .. } => {
+                    panic!()
+                } // FIXME: impl TryFrom
+            };
+
+            let mint_client = client
+                .get_module_client::<MintClientModule>(LEGACY_HARDCODED_INSTANCE_ID_MINT)
+                .unwrap();
+
+            let client_input = mint_client
+                .create_input(
+                    &mut dbtx.with_module_prefix(LEGACY_HARDCODED_INSTANCE_ID_MINT),
+                    operation_id,
+                    amount,
+                )
+                .await?;
```

Comment:
You actually don't need this, `finalize_and_submit_transaction` will add funds as needed.

https://github.com/fedimint/fedimint/blob/fccd7396a5f294e61dc503b5446911823d74e720/fedimint-cli/src/ng.rs#L66-L72

Reply:
This is fixed